docs/1.9/03-Tutorials2/02-Build-GraphQL-Servers/01-Development/01-Build-a-GraphQL-Server-with-Prisma.md

```diff
@@ -406,7 +406,7 @@ The implementation of your GraphQL server is now done and you can work with it i
 In your terminal, start the server using the following command:
 
 ```sh
-node src/index.js
+node --harmony src/index.js
```

Comment:
I think a better suggestion would be to add a key with `engine` version in `package.json` that specifies the min node version.

https://docs.npmjs.com/files/package.json#engines

Reply:
As you wish, but 8.2.1 is the latest (recommended) version per node's site. And it works, but requires the --harmony flag. I can see your point too though. Maybe both since neither one on it's own is a complete solution?